config/confighttp/confighttp.go

```diff
@@ -183,11 +183,6 @@ func (hcs *HTTPClientSettings) ToClient(host component.Host, settings component.
 	}, nil
 }
 
-// Deprecated: [v0.57.0] use ToClient.
-func (hcs *HTTPClientSettings) ToClientWithHost(host component.Host, settings component.TelemetrySettings) (*http.Client, error) {
```

Comment:
Is this related to component stability helpers or should removing this deprecated function be a separate PR?

Reply:
Removed, will do a separate PR.